services/kuber/templates/cluster-autoscaler.goyaml

```diff
@@ -12,16 +15,19 @@ kind: Deployment
 metadata:
   name: autoscaler-{{ .ClusterID }}
   labels:
-    app: autoscaler-{{ .ClusterID }}
+    app.kubernetes.io/part-of: claudie
+    app.kubernetes.io/name: cluster-autoscaler-{{ .ClusterID }}
```

Comment:
I think this shouldn't be templated, I'd go with `app.kubernetes.io/name: cluster-autoscaler` for everything

Reply:
There are 2 deployments. One for cluster-autoscaler and one for autoscaler-adapter. If we use app.kubernetes.io/name: cluster-autoscaler label on both of them then the deployments will also select each others pods. We need a separate label for each of them, differentiating the pods of cluster-autoscaler and autoscaler-adapter.